apps/condo/domains/common/components/PostMessageProvider/types.ts

```diff
@@ -15,13 +15,15 @@ export const COMMON_ERROR_PREFIX = 'CondoWebAppCommonError' as const
 export type RequestParamsMap = {
     [Method in BridgeRequestMethodsName]: BridgeRequestParams<Method>
 } & {
-    CondoWebSendAnalyticsEvent: AnalyticsParams
+    CondoWebSendAnalyticsEvent: AnalyticsParams,
+    CondoWebSetActiveCall: { isActiveCall: boolean, connectedTickets: string[] }
 }
 
 export type HandlerResultsMap = {
     [Method in BridgeResponseMethodsName]: BridgeResponseData<Method>
 } & {
-    CondoWebSendAnalyticsEvent: { sent: boolean }
+    CondoWebSendAnalyticsEvent: { sent: boolean },
+    CondoWebSetActiveCall: void
```

Comment:
Why there's no indication, that call handling was successful or not or anything like that?